config/stp.py

```diff
@@ -465,6 +778,7 @@ def check_if_stp_enabled_for_vlan(ctx, db, vlan_name):
         ctx.fail("STP is not enabled for VLAN")
 
 
+#config spanning_tree vlan enable <vlan-id>
```

Comment:
This command is valid only for PVST. Please add check.

Reply:
I have addressed this comment, please review